Header header

float64[] some_x # variable-sized array of floats
float64[] some_y # variable-sized array of floats

